src/style.css

```diff
@@ -1,3 +1,72 @@
-body {
+html {
+  font-family: "Roboto", sans-serif;
+  --normal-color: #000;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body,
+ul,
+li,
+header {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  --active-color: #00acdc;
```

Comment:
good practice to declare variables inside the [:root](https://www.w3schools.com/css/css3_variables.asp) selector
```css
:root {
 --active-color: #00acdc;
 --normal-color: #000;
}
```